### PR TITLE
Add more annotations to test_cholesky_solve_op.py

### PR DESCRIPTION
**Pull request by zhiboniu**
### PR types

Others
### PR changes

Docs
### Describe

Add more annotations to test_cholesky_solve_op.py, make it an example in hackson guide.
test=document_fix

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py [hunk 1]
```diff
@@ -54,6 +56,7 @@ def scipy_cholesky_solution(X, B, upper=True):
     return scipy.linalg.cho_solve(K, B)
 
 
+#boardcast function used by cholesky_solve
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py]: boardcast-》broadcast，全文有好几个同样的笔误

File: python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py [hunk 2]
```diff
@@ -90,18 +94,21 @@ def scipy_cholesky_solution_batch(bumat, bB, upper=True):
...
     case 1
     """
 
+    #test condition set
     def config(self):
         self.y_shape = [15, 15]
         self.x_shape = [15, 5]
         self.upper = False
-        self.dtype = np.float64
+        self.dtype = np.float64  #cholesky_solve Op only support float type, elsewise please add other types
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py]: Here cholesky_solve Op only supports float64/float32 type, please check others if Op supports more types.

File: python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py (hunk 1 above)
Comment (zhiboniu) [reply python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py]: done

File: python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py (hunk 2 above)
Comment (zhiboniu) [reply python/paddle/fluid/tests/unittests/test_cholesky_solve_op.py]: done

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**